python/elder/Cursoemvideo/ex003.py

```diff
@@ -0,0 +1,8 @@
+# n1 = int(input('Digite um valor: '))
+# n2 = int(input('Digite outro valor: '))
+# s = n1 + n2
+# print(f'A soma entre {n1} e {n2} é igual a {s}')
+
+n1 = int(input('Digite um valor: '))
+n2 = int(input('Digite outro valor: '))
+print(f'A soma entre {n1} e {n2} é igual a {n1 + n2}')
```

Comment:
```suggestion
print(f'A soma entre {n1} e {n2} é igual a {n1 + n2}')
```

Esse errinho que o github indica sinaliza que falta um caractere de quebra de linha no final.